dataService.js

```diff
@@ -0,0 +1,78 @@
+// Maps
+let productMap = {};
+let categoryMap = {};
+let combinedProductMap = null;
+
+// Method to create map of products
```

Comment:
Better to use multiline comments for method comments.
```
/*
 * comment
 */
```

Reply:
Done!